cmd/nerdctl/run_network_linux_test.go

```diff
@@ -513,26 +513,19 @@ func TestSharedNetworkStack(t *testing.T) {
 
 	containerNameJoin := testutil.Identifier(t) + "-nework"
 	defer base.Cmd("rm", "-f", containerNameJoin).AssertOK()
-	cmd := base.Cmd("run",
+	base.Cmd("run",
 		"-d",
 		"--name", containerNameJoin,
 		"--network=container:"+containerName,
 		testutil.CommonImage,
-		"sleep", "infinity")
-	cmd.AssertOK()
-
-	base.Cmd("exec", containerNameJoin, "wget", "-qO-", "http://127.0.0.1:80").
-		AssertOutContains(testutil.NginxAlpineIndexHTMLSnippet)
+		"sleep", "infinity").AssertOK()
 
-	base.Cmd("restart", containerName).AssertOK()
-	base.Cmd("restart", containerNameJoin).AssertOK()
 	base.Cmd("exec", containerNameJoin, "wget", "-qO-", "http://127.0.0.1:80").
 		AssertOutContains(testutil.NginxAlpineIndexHTMLSnippet)
 
 	base.Cmd("restart", containerName).AssertOK()
```

Comment:
How about splitting the restart action to stop and start shifting to keep the same as the containerNameJoin?

Reply:
Conversely, why not keep `restart containerNameJoin`?

Because in fact we want to verify that `start` makes the container join the shared network instead of `stop`.

Reply:
LGTM

Reply:
guys, I didn't understand why we keep restart here?